src/parser/core/modules/Combos.tsx

```diff
@@ -18,16 +18,30 @@ const GCD_TIMEOUT_MILLIS = 15000
 const ISSUE_TYPENAMES = {
 	uncomboed: <Trans id="core.combos.issuetypenames.uncomboed">Uncomboed</Trans>,
 	combobreak: <Trans id="core.combos.issuetypenames.combobreak">Broken Combo</Trans>,
+	failedcombo: <Trans id="core.combos.issuetypenames.failed">Missed or Invulnerable</Trans>,
 }
 
-export interface ComboEvent extends AbilityEvent {
+export interface ComboEvent extends AoeEvent {
 	type: 'combo'
 }
 
+interface Hit {
+	id: number
+	instance: number
+	times: number
+	amount: number
+}
+
+export interface AoeEvent extends AbilityEvent {
```

Comment:
be good to add a note here and in the aoe module that this should be over there

Reply:
Updates made.